lib/screens/mainScreen.js

```diff
@@ -1,29 +1,58 @@
 import React, { Component } from 'react';
-import { StyleSheet, View, Text, TextInput } from 'react-native';
+import { StyleSheet, View, Text, Alert, Button, ScrollView } from 'react-native';
 import Dropdown from './dropdown';
 import Maps from './maps';
+import DepartArriveButtonGroup from './departArriveButtonGroup';
+import DateTimePicker from './dateTimePicker';
+import AddLocation from './addLocation';
 
 export default MainScreen = () => {
+
     return (
         <View style={styles.mainScreenStyle}>
+
+            <AddLocation/>
+
             <View style={styles.dropdownContainer}>
-                <Dropdown placeholder='Start Location'/>
+                <DepartArriveButtonGroup />
             </View>
+
             <View style={styles.dropdownContainer}>
-                <Dropdown placeholder='End Location'/> 
+                <DateTimePicker />
             </View>
+
+            <View style={styles.dropdownContainer}>
+                <Button
+                    title='Get Route'
+                    onPress={() => {
+                        Alert.alert('Confirm',
+                            'Get route for inputted stops?',
+                            [
+                                { text: 'OK' },
+                                //onPress: () => },
```

Comment:
```suggestion
                                { text: 'OK', onPress: () => {} },
```
Would recommend to just hook up the onPress to an empty function for now, makes it easier later on for when we implement the press handlers